pkg/reconciler/testing/v1/subscription.go

```diff
@@ -282,3 +284,31 @@ func WithSubscriptionReply(gvk metav1.GroupVersionKind, name, namespace string)
 		}
 	}
 }
+
+func WithSubscriptionOptionOIDCIdentityCreatedSucceeded() SubscriptionOption {
```

Comment:
```suggestion
func WithSubscriptionOIDCIdentityCreatedSucceeded() SubscriptionOption {
```
can we rename it to keep it a bit shorter? (same for the other `WithSubscriptionOptionOIDCIdentityXYZ()` functions)